backend/xivdpscalc/pps/sch.py

```diff
@@ -105,3 +105,13 @@ def mp_consumed_per_cycle(cls, character_stats, caster_tax, succ, adlo, ed, rez,
         # Energy Drain
         result -= 500 * ed * cycle / 60
         return result
+
+    # TODO: it wasn't implement in Pps or HealerPps, so clearly it needs to be implemented at
```

Comment:
If you think it's out of scope for the PR, feel free to resolve without action, but it should be fine to have this call the existing SchPps.get_pps with the default values

Reply:
The default for the parent class does a `raise NotImplementedError`, so it felt like inheriting classes should specifically override this (hence why I stubbed it out here and did a TODO)

Reply:
i'm with accel on this one -- as part of the cleanup we should be just implementing that method. and the right thing to do is to use the get_pps existing implementation.

right now the todo is confusing because we clearly have that get_pps function and it's not like it's much effort to use that code.

Reply:
No problem for me – I reverted SchPps to it's previous non-abstract implementation (that raises NotImplemented).

Reply:
good enough for me. at some point someone should actually implement it but that's a problem for future us i guess...